src/convertToObject.js

```diff
@@ -11,8 +11,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  let stringCSS = sourceString.split(';');
+  const result = {};
+
+  stringCSS = stringCSS.map(x => x.replace(/\n|\s\s+/g, ''));
+  stringCSS = stringCSS.filter(x => x.length > 1);
```

Comment:
don't use letters as a names

Reply:
no need to use filter just check if an element exists before adding to an object and use a trim.